Source/JavaScriptCore/jit/JITOpcodes.cpp

```diff
@@ -1652,6 +1637,29 @@ void JIT::emit_op_create_this(const JSInstruction* currentInstruction)
     emitPutVirtualRegister(bytecode.m_dst, jsRegT10);
 }
 
+void JIT::emit_op_to_this(const JSInstruction* currentInstruction)
+{
+    auto bytecode = currentInstruction->as<OpToThis>();
+    emitGetVirtualRegister(bytecode.m_srcDst, jsRegT10);
+
+    Jump isCell = branchIfCell(jsRegT10);
+    addSlowCase(branchIfNotOther(jsRegT10, regT3));
+
+#if USE(JSVALUE32_64)
+    move(TrustedImm32(JSValue::CellTag), jsRegT10.tagGPR());
+#endif
```

Comment:
We should do `boxCell` before setting `emitPutVirtualRegister`.

Reply:
Done.